### DISCUSSION TITLE
multi level expand

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#5819] [Q&A] [answered]
```
 const records = await pb.collection('auctions').getFullList({
            sort: '-created',
            expand:'catalougs.Factory,catalougs.Warehose.catalougs.Liquor_rating'
        });
```

here catalougs.Factory expand properly rest of the item not expanding.is my approach wrong?

![Screenshot (625)](https://github.com/user-attachments/assets/b0384a51-3eff-47f7-80b9-76f274988249)

![Screenshot (624)](https://github.com/user-attachments/assets/6c2e9c30-54f9-46cb-97ed-bb606a693798)

**Comment by chnfyi** [comment, marked as answer]
There is a potential typo in your expand string. You can try it.

```javascript
 const records = await pb.collection('auctions').getFullList({
            sort: '-created',
            expand:'catalougs.Factory,catalougs.Warehose,catalougs.Liquor_rating'
        });
```

**Comment by tigawanna** [comment]
if you're using typescript consider 
https://github.com/david-plugge/typed-pocketbase for auto completion
